<!DOCTYPE html>

<html class="gr__princeton_edu">

<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
		
		<title>Xin Wang - Home</title><meta name="description" content="Website for Xin Wang">
	
	<meta name="keywords" content="Computational biology, machine learning, deep learning">

	<meta name="viewport" content="width=device-width, initial-scale=1.0">

	<link rel="stylesheet" type="text/css" href="./data/main_style.css">
	<link href="./data/css" rel="stylesheet">

	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script type="text/javascript" async="" src="./data/analytics.js"></script><script async="" src="./data/js"></script>
	<script>
	  window.dataLayer = window.dataLayer || [];
	  function gtag(){dataLayer.push(arguments);}
	  gtag('js', new Date());

	  gtag('config', 'UA-68992915-1');
	</script>
</head>


<!--- Header -->
<div id="header">
	<label for="nav-trigger" class="nav-trigger hamburger">
		<span class="open-btn">
			<span class="mobile"></span>
			<span class="mobile"></span>
			<span class="mobile"></span>
		</span>
	</label>
	<div id="sitename">
		<span class="wsite-logo">

		<a href="http://flyboyleo.github.io">

		    <span id="wsite-title" style="position:relative; top:5px">Xin Wang</span><img src="./data/profile.png" style="border-radius: 50%; width: 6%;margin-left: 2%; vertical-align: middle">

		</a>

		</span>
	</div>
</div>

<div id="wrapper">
	<div class="bg-wrapper">
		<div id="navigation"><ul class="wsite-menu-default">
	<li id="active" class="wsite-menu-item-wrap">
		<a href="http://flyboyleo.github.io/#home" class="wsite-menu-item">
			Home
		</a>
		
	</li>
	<li id="pg191988064268116182" class="wsite-menu-item-wrap">
		<a href="http://flyboyleo.github.io/#pub" class="wsite-menu-item">
			Publication
		</a>
		
	</li>
</ul>
</div>
			

<!--- Bio -->

<div id="content-wrapper">
	<div id="content"><div id="wsite-content" class="wsite-elements wsite-not-footer">

<h1 class="wsite-content-title" style="text-align:left;"><font size="5">Bio</font></h1>

<div class="wsite-spacer" style="height:10px;"></div>
<div class="wsite-spacer" style="height:10px;"></div>

<div class="paragraph" style="text-align:left;">
	<span>
        Hi. This is Xin from the Department of Biomedical Informatics (DBMI) of Harvard Medical School. I recently joined Peter Kharchenko lab for a Consortium effort project and spatial deep learning project with single-cell analysis.
		
		
	Before coming to Boston, I spent two years at <a href="https://www.northwestern.edu/">Northwestern University</a> Chicago campus. My previous postdoctoral research focused on the mechanism of gene regulations via <a href="https://www.nature.com/articles/s41576-019-0093-7">single-cell analysis</a> of cancers and machine/<a href="https://www.nature.com/articles/d41586-018-02174-z">deep learning</a>.

        <br><br>

		I received my Ph.D. in Bioinformatics and Computational Biology at <a href="https://www.usnews.com/education/best-global-universities/university-of-new-south-wales-502434">the University of New South Wales</a> in Sydney, NSW, Australia under <a href="https://www.sbms.hku.hk/staff/joshua-wing-kei-ho">Joshua Ho</a> in 2018. My Ph.D. research focused on developing machine learning models for <a href="https://link.springer.com/article/10.1007%2Fs00439-013-1413-1">the grammar of transcriptional regulation</a> and <a href="https://www.nature.com/articles/nmeth.1673">chromatin state</a> study.

		<br><br>

		Before coming to UNSW, I got my Bachelor’s degree at <a href="https://www.usnews.com/education/best-global-universities/sun-yat-sen-university-506062">Sun Yat-sen University</a> in Guangzhou, China in 2009. I also got my Master of Research degree in 2013 under <a href="https://www.researchgate.net/profile/Zhongsheng_Sun">Dr. Zhongsheng Sun</a> who is a PI in the Beijing Institutes of Biological Science, Chinese Academy of Science(CAS). After master graduation, I was in <a href="https://en.wikipedia.org/wiki/Biocant">Biocant Institute</a> (@Portugal) for one year as a Bioinformatician.

        <br><br>
        Contact: xin.leo.wang@outlook.com 

	</span>
</div>

<div class="wsite-spacer" style="height:50px;"></div>



<!--- Research interests-->

<h1 class="wsite-content-title" style="text-align:left;"><font size="5">Research interests</font></h1>

<div class="wsite-spacer" style="height:10px;"></div>
<div class="wsite-spacer" style="height:10px;"></div>
<div class="paragraph" style="text-align:left;">


<h2 class="wsite-content-title" style="text-align:left;"><font size="4">Predict the functional impact of genetic variants via ultra-deep neural networks such as Resnet</font></h2>
<span> 
    Predict the functional impact of non-coding genetic variants is crucial for related diseases' diagnosis and clinical applications. Based on the sate-of-the-art <a href="https://en.wikipedia.org/wiki/Residual_neural_network">ResNet</a> (<a href="https://arxiv.org/abs/1512.03385">Residual neural network paper</a>), a recent published work <a href="https://www.sciencedirect.com/science/article/pii/S0092867418316295">"SpliceAI"</a> shows unbelievable accuracy on predicting variants caused splicing. I'm quite familiar with SpliceAI as established and re-ran it on our current GPU workstation. It would be a good combination for Resnet structure and other mutation effect predictions such as polyadenylation (manuscript in preparation) or other diseases mutation study with/without the related CRISPR/cas9 cell-line screening data.
</span>
<br><br>
   

<h2 class="wsite-content-title" style="text-align:left;"><font size="4">Single-cell RNA-seq anaylsis on diseases (e.g. cancers)</font></h2>
<span>
     Single-cell RNA-seq technology now becomes more and more popular and useful in disease studies as well as related immunotherapy studies. I have the experience on analyzing single-cell RNA-seq data generated from both SMART-seq2 protocol (our chemoresistance index project with <a href="https://biophotonics.bme.northwestern.edu">Vadim Backman lab</a>) and 10X Genomics protocol (a preliminary exploration analysis collaborated with <a href="https://struhl.med.harvard.edu">Kevin Struhl lab</a>). My single-cell RNA-seq analysis on the SMART-seq2 data revealed interesting biology in chemoresistance (manuscript in preparation).
</span>
<br><br>

    
<h2 class="wsite-content-title" style="text-align:left;"><font size="4">Discovery of cell-type specific DNA motif grammar in cis-regulatory elements</font></h2>
<span>
    I developed a Random Forest (RF) based approach to build a multi-class classifier to predict the cell-type specificity of a TF binding site given its motif content. I applied this RF classifier to two published ChIP-seq datasets of TF (TCF7L2 and MAX) across multiple cell types. Using cross-validation, I show that motif combinations alone are indeed predictive of cell types. I also present a rule mining approach to extract the most discriminatory rules in the RF classifier, thus allowing us to discover the underlying cell-type specific motif grammar.
</span>
</div>



<!--- Publications -->

<div class="wsite-spacer" style="height:50px;"></div>

<h1 id="pub" class="wsite-content-title" style="text-align:left;"><font size="5">Publications</font> <a href="https://scholar.google.com/citations?hl=en&user=nt-2WnMAAAAJ&view_op=list_works&sortby=pubdate"><font size="4">(Google Scholar)</font></a></h1>

<div class="paragraph" style="text-align:left;"> (* indicate co-first authors):<br><br>

    <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-4340-z"><strong>
    Discovery of cell-type specific DNA motif grammar in cis-regulatory elements using random Forest
    </strong></a>
    <br>
    ​<strong>Xin Wang</strong>, Peijie Lin, Joshua W. K. Ho
    <br>
    <i>
    <strong>BMC Genomics</strong>, 2018
    </i>
    <br><br>

    

    <a href="https://www.nature.com/articles/s41598-018-19855-4"><strong>
	Biologically active constituents of the secretome of human W8B2+ cardiac stem cells
    </strong></a>
    <br>
    ​Shuai Nie*, <strong>Xin Wang*</strong> , Priyadharshini Sivakumaran*, ...,Joshua W. K. Ho, Nicholas Williamson, Shiang Y. Lim
    <br>
    <i>
    <strong>Scientific Reports</strong>, 2018
    </i>
    <br><br>



    <a href="https://academic.oup.com/nar/article/45/9/e67/2918642"><strong>
    PBrowse: a web-based platform for real-time collaborative exploration of genomic data
    </strong></a>
    <br>
    ​Peter S. Szot, Andrian Yang, <strong>Xin Wang</strong>, Chirag Parsania, Uwe Röhm, Koon Ho Wong, Joshua W. K. Ho,
    <br>
    <i>
    <strong>Nucleic Acids Research</strong>, 2017
    </i>
    <br><br>



    <a href="https://eprints.victorchang.edu.au/400/"><strong>                                                                             
    Epigenomic analysis of chromatin organization and DNA methylation (Book Chapter)
    </strong></a>
    <br>
    ​<strong>Xin Wang</strong>, Helen M. McCormick, Djordje Djordjevic, Eleni Giannoulatou, Catherine M. Suter, Joshua WK Ho
    <br>
    <i>
    <strong>Computational Biology and Bioinformatics: Gene Regulation</strong>,
    </i>
    CRC Press, 2016
    <br><br>



    <a href="http://www.jbc.org/content/289/20/14225.short"><strong>
    Ras-induced epigenetic inactivation of the RRAD (Ras-related associated with diabetes) gene promotes glucose uptake in a human ovarian cancer model
    </strong></a>
    <br>
    (The bioinformatics analysis part of this paper are also mostly in <a href="https://github.com/flyboyleo/flyboyleo.github.io/blob/master/data/%E7%8E%8B%E9%91%AB%E6%AF%95%E4%B8%9A%E8%AE%BA%E6%96%87_%E9%81%97%E4%BC%A0%E5%AD%A6_2010010022.pdf">my Master Thesis</a>)
    <br>
    ​Yan Wang, Guiling Li, Fengbiao Mao, Xianfeng Li, Qi Liu, Lin Chen, Lu Lv,  <strong>Xin Wang</strong>, Jinyu Wu, Wei Dai, Guan Wang, Enfeng Zhao, Kai-Fu Tang and Zhong Sheng Sun
    <br>
    <i>
   	<strong>The Journal of Biological Chemistry</strong>, 2014
    </i>
    <br><br>


    <a href="https://www.tandfonline.com/doi/full/10.4161/rna.25193"><strong>
    mirTools 2.0 for non-coding RNA discovery, profiling, and functional annotation based on high-throughput sequencing
    </strong></a>

    <br>
    ​Jinyu Wu, Qi Liu, <strong>Xin Wang</strong>, Jiayong Zheng, Tao Wang, Mingcong You, Zhong Sheng Sun and Qinghua Shi
    <br>
    <i>
    <strong>RNA Biology</strong>, 2013
    </i>
    <br><br>



    <a href="https://academic.oup.com/nar/article/45/9/e67/2918642"><strong>
    Exome-assistant: A rapid and easy detection of disease-related genes and genetic variations from exome sequencing
    </strong></a>
    <br>
    ​Qi Liu, Enjian Shen, Qingjie Min, Xueying Li, <strong>Xin Wang</strong>, Xianfeng Li, Zhong Sheng Sun & Jinyu Wu 
    <br>
    <i>
    <strong>BMC Genomics</strong>, 2012
    </i>
    <br><br>


<!--- Submitted paper -->

<br>

<h1 id="pub" class="wsite-content-title" style="text-align:left;"><font size="5">Submitted paper</font></h1>
<br>


    <strong>
    Antibody targeting tumor-derived soluble NKG2D ligand sMIC reprograms NK cell function and enhances melanoma response to PD1 blockade therapy
    </strong>
    <br>
    ​Fahmin Basher, <strong>Xin Wang</strong>, Payal Dhar, Zhe Ji, Jennifer Wu
    <br>
    <i>
    Submitted to <strong>Journal for ImmunoTherapy of Cancer</strong>
    </i>
    <br><br>


<!--- Software -->

<br>

<h1 id="pub" class="wsite-content-title" style="text-align:left;"><font size="5">Software</font></h1>
<br>

    <a href="https://github.com/flyboyleo/POMP--a-Powerful-Motif-discovery-Pipeline"><strong>
    POMP: a-Powerful-Motif-discovery-Pipeline (from my 2018 Random Forest paper)
    </strong></a>
    <br>
    ​<strong>Xin Wang</strong>, Joe Godbehere
    <br>
    <br>


<!--- Posters -->

<br>

<h1 id="pub" class="wsite-content-title" style="text-align:left;"><font size="5">Posters</font></h1>
<br>
    <strong>
    Single-cell RNA-seq reveals transcriptional heterogeneities in oxidative stress response and metabolic processes mediating chemoresistance
    </strong>
    <br>
    ​<strong>Xin Wang</strong>, Vadim Backman, Zhe Ji
    <br>
    <a href="https://physicalgenomics.northwestern.edu/news-events/symposium-on-physical-genomics.html">Symposium on Physical Genomics, 2019</a>
    <br>
    <br>

    <a href="https://www.iscb.org/cms_addon/conferences/ismb2016/posterlist.php?cat=D"><strong>
    Comprehensive analysis of chromatin landscape in filamentous fungus <i>Aspergillus nidulans</i>
    </strong></a>
    <br>
    ​<strong>Xin Wang</strong>, Djordje Djordjevic, Zhengqiang Miao, Chirag Parsania, Kaeling Tan, Joshua W. K. Ho, Koon Ho Wong
    <br>
    Intelligent Systems for Molecular Biology (ISMB), 2016
    <br>
    <br>


<!--- Talks -->

<br>

<h1 id="pub" class="wsite-content-title" style="text-align:left;"><font size="5">Talks</font></h1>
<br>
    <strong>
    Single-Cell RNA-Seq Reveals Transcriptional Heterogeneities Mediating Chemoresistance
    </strong>
    <br>
    Department of Pharmacology annual retreat, Northwestern University (May 2nd, 2019)
    <br>
    <br>

    <strong>
    Discovery of cell-type specific DNA motif grammar in cis-regulatory elements using Random Forest
    </strong>
    <br>
    International Conference on Bioinformatics (InCoB) 2017
    <br>
    <br>

    <strong>
    Comprehensive analysis of chromatin landscape in filamentous fungus <i>Aspergillus nidulans</i>
    </strong>
    <br>
    Oral presentation at Student Council Symposium of ISMB 2016
    <br>
    <br>


<!--- Honors/Awards -->

<br>

<h1 id="pub" class="wsite-content-title" style="text-align:left;"><font size="5">Honors/Awards</font></h1>
<br>

    <strong>
    Tuition Fee Scholarship (TFS) plus a Research Stipend (2014 - 2018)
    </strong>
    <br>
    <br>

    <strong>
    The Postgraduate Research Student Support (PRSS) Conference Travel Funds (2016)
    </strong>
    <br>
    <br>

    <a href="https://www.apbionet.org/incob-travel-fellowship/"><strong>
    International Conference on Bioinformatics (InCoB) Travel Fellowship (2017)
    </strong></a>
    <br>
    <br>

    <strong>
    Nomination for the Higher Degree Research Thesis Award of UNSW (2018)
    </strong>
    <br>
    <br>
    <br>
    <br>
    <br>
    <br>





<i>Last update: March 6, 2020</i>




	
